bin/cml/pr/create.e2e.test.js

```diff
@@ -10,8 +10,8 @@ describe('CML e2e', () => {
       Manage pull requests
 
       Commands:
-        cml.js pr create <glob path...>  Create a pull request with the specified
-                                         files
+        cml.js pr create [glob path...]  Create a pull request, optionally committing
+                                         the specified files first.
```

Comment:
don't mind about the words; but mainly would like the period removed for consistency with other commands.

```suggestion
        cml.js pr create [glob path...]  Create a pull request (committing any given
                                         paths first)
```